test/main.js

```diff
@@ -7,11 +7,28 @@
 require.config({
   baseUrl: '..',
   paths: {
-    'openpgp': 'dep/chrome/openpgpjs/dist/openpgp.js'
+    mvelo: 'common/ui/mvelo',
+    parser_rules: 'common/dep/wysihtml5/js/advanced_parser_rules',
+    dompurify: 'bower_components/dompurify/src/purify',
+    jquery: 'bower_components/jquery/dist/jquery',
+    openpgp: 'dep/chrome/openpgpjs/dist/openpgp',
+    'lib/json-loader': 'chrome/lib/json-loader',
+    'common/lib-mvelo': 'chrome/lib/lib-mvelo',
+    'emailjs-mime-builder': 'build/chrome/lib/emailjs-mime-builder',
+    'mailreader-parser': 'build/chrome/lib/mailreader-parser',
+    'emailjs-mime-codec': 'build/chrome/lib/emailjs-mime-codec',
+    'emailjs-mime-types': 'build/chrome/lib/emailjs-mime-types',
+    'emailjs-punycode': 'build/chrome/lib/emailjs-punycode',
+    'emailjs-addressparser': 'build/chrome/lib/emailjs-addressparser',
+    'emailjs-mime-parser': 'build/chrome/lib/emailjs-mime-parser',
+    'emailjs-stringencoding': 'build/chrome/lib/emailjs-stringencoding'
   },
```

Comment:
... we just needed to wire up the runtime dependencies using the require.js config.